### DISCUSSION TITLE
怎么返回 key_cls 参数

### DISCUSSION THREAD

**Discussion by q465414859** [PaddlePaddle/PaddleOCR#12270] [Q&A]
框选数据的工具中发现有个参数 key_cls 训练后想在识别中返回这个数据，要怎么操作？

**Comment by q465414859** [comment]
> 框选数据的工具中发现有个参数 key_cls 训练后想在识别中返回这个数据，要怎么操作？

![1714380782338](https://github.com/PaddlePaddle/PaddleOCR/assets/26251132/0c0bd613-f925-4d1a-b4e4-15865ddf7405)
![1714380882097](https://github.com/PaddlePaddle/PaddleOCR/assets/26251132/390df187-929c-423a-b24a-e20c134bf3f1)

**Comment by SWHL** [comment]
请移步这里提问：https://github.com/PFCCLab/PPOCRLabel

**Comment by lemonit-eric-mao** [comment]
这个问题要怎么解决呢？有没有人能指导一下？我也有同样的问题。
